x-pack/plugins/triggers_actions_ui/public/common/get_edit_connector_flyout.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React, { lazy, Suspense } from 'react';
+import { EuiLoadingSpinner, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { ConnectorEditFlyoutProps } from '../application/sections/action_connector_form/connector_edit_flyout';
+
+export const getEditConnectorFlyoutLazy = (props: ConnectorEditFlyoutProps) => {
+  const ConnectorEditFlyoutLazy = lazy(
+    () => import('../application/sections/action_connector_form/connector_add_flyout')
```

Comment:
Should this be `edit`?

```suggestion
    () => import('../application/sections/action_connector_form/connector_edit_flyout')
```